web-server/v0.4/src/utils/parse.js

```diff
@@ -224,6 +227,10 @@ export const parseClusteredIterations = (clusteredIterations, clusterLabels, sel
               return false;
             })
           ];
+        const percentage =
+          ((clusterObject[iteration] - clusterObject[0]) / clusterObject[iteration]) * 100;
```

Comment:
The calculation of percent difference is missing one aspect of the formula where the average of both numbers is used as the divisor. The formula for calculating percent difference is as follows:
`|x1 - x2|/((x1 + x2)/2) * 100`